diff --git a/server/Makefile b/server/Makefile
index b73bee82d3..7fcc0efaf2 100644
--- a/server/Makefile
+++ b/server/Makefile
@@ -62,7 +62,7 @@ MMCTL_LDFLAGS += -X "$(MMCTL_PKG).buildDate=$(MMCTL_BUILD_DATE)"
 
 # Enterprise
 BUILD_ENTERPRISE_DIR ?= ../../enterprise
-BUILD_ENTERPRISE ?= true
+BUILD_ENTERPRISE ?= false
 BUILD_ENTERPRISE_READY = false
 BUILD_TYPE_NAME = team
 BUILD_HASH_ENTERPRISE = none
diff --git a/server/channels/app/limits.go b/server/channels/app/limits.go
index 71032d4beb..14f0551a3e 100644
--- a/server/channels/app/limits.go
+++ b/server/channels/app/limits.go
@@ -12,8 +12,8 @@ import (
 )
 
 const (
-	maxUsersLimit     = 10_000
-	maxUsersHardLimit = 11_000
+	maxUsersLimit     = 10_000_000
+	maxUsersHardLimit = 11_000_000
 )
 
 func (a *App) GetServerLimits() (*model.ServerLimits, *model.AppError) {
