diff --git a/server/Makefile b/server/Makefile
index e58c53e9c4..ce8494442d 100644
--- a/server/Makefile
+++ b/server/Makefile
@@ -59,7 +59,7 @@ MMCTL_LDFLAGS += -X "$(MMCTL_PKG).buildDate=$(MMCTL_BUILD_DATE)"
 
 # Enterprise
 BUILD_ENTERPRISE_DIR ?= ../../enterprise
-BUILD_ENTERPRISE ?= true
+BUILD_ENTERPRISE ?= false
 BUILD_ENTERPRISE_READY = false
 BUILD_TYPE_NAME = team
 BUILD_HASH_ENTERPRISE = none
diff --git a/server/channels/app/limits.go b/server/channels/app/limits.go
index 5697cd224a..d54ee7fb6e 100644
--- a/server/channels/app/limits.go
+++ b/server/channels/app/limits.go
@@ -12,10 +12,10 @@ import (
 )
 
 const (
-	maxUsersLimit     = 10_000
-	maxUsersHardLimit = 11_000
+	maxUsersLimit     = 10_000_000
+	maxUsersHardLimit = 11_000_000
 
-	maxPostLimit = 5_000_000
+	maxPostLimit = 5_000_000_000_000
 )
 
 func (a *App) GetServerLimits() (*model.ServerLimits, *model.AppError) {
